MeshLib/MeshEditing/AddLayerToMesh.cpp

```diff
@@ -114,23 +114,28 @@ MeshLib::Mesh* addLayerToMesh(MeshLib::Mesh const& mesh, double thickness,
 			extrudeElement(subsfc_nodes, *sfc_elements[k], subsfc_sfc_id_map)
 		);
 
-	MeshLib::Properties subsfc_props (mesh.getProperties());
-	boost::optional<MeshLib::PropertyVector<int> &> opt_materials(
-		subsfc_props.getPropertyVector<int>("MaterialIDs")
+	boost::optional<MeshLib::PropertyVector<int> const&> opt_materials(
+		mesh.getProperties().getPropertyVector<int>("MaterialIDs")
 	);
-	if (!opt_materials) {
-		ERR("Can not set material properties for new layer");
```

Comment:
Now there is no error message anymore when `if (!opt_materials)` holds. Or is that normal behaviour?